keras/backend/tensorflow_backend.py

```diff
@@ -944,34 +930,89 @@ def _normalize_axis(axis, ndim):
 
 def max(x, axis=None, keepdims=False):
     '''Maximum value in a tensor.
+
+    # Arguments
+        x: A tensor or variable.
+        axis: A list of integer. Axes to find maximum values.
```

Comment:
It should be described as a single integer, for consistency with the Theano backend (although we don't enforce this on the TF side).